utopian-io/v2.utopian.io
# View article page

### Verify that
Git checklist

Front end checklist

API checklist

Tests and comments checklist

### **Test plan**

_View an article_

* Go to http://localhost:8080/en/articles/gregory/utopian-v2-task-article-module-add-tags

**Expected result:** You should

* Check that the view count is increasing (technically it won't because we all have the same local IP)
* Author and collaborators are links that points to /@username
* you're the author you should see the edit icon

NB: Design of the html supported tags in the article content will be handled later
